internal/librarian/tag_and_release.go

```diff
@@ -258,3 +329,58 @@ func (r *tagAndReleaseRunner) replacePendingLabel(ctx context.Context, p *github
 	}
 	return nil
 }
+
+// updateVersionAndBody is a helper function to update a map tracking release information for each library.
+func updateVersionAndBody(idToVersionAndBody map[string]*releaseSection, library, commitType, title, message, version string) {
+	vab, ok := idToVersionAndBody[library]
+	if !ok {
+		idToVersionAndBody[library] = &releaseSection{
+			typeToMessages: map[string][]string{
+				commitType: {message},
+			},
+			version: version,
+			title:   title,
+		}
+
+		return
+	}
+
+	vab.typeToMessages[commitType] = append(vab.typeToMessages[commitType], message)
+	if version == "" {
+		version = vab.version
+	}
+	vab.version = version
+	if title == "" {
+		title = vab.title
+	}
+	vab.title = title
+}
+
+func updateReleaseBody(body map[string][]string, title string) string {
```

Comment:
nit: can you add godoc here. Reading the signature it is hard to tell what is going one. It is titled updated but appears to return a new string and it is not obvious what the key/values of the map are.